test/src/unit-class_lexer.cpp

```diff
@@ -37,10 +37,10 @@ using nlohmann::json;
 namespace
 {
 // shortcut to scan a string literal
-json::lexer::token_type scan_string(const char* s);
```

Comment:
Please leave this in - I added the declaration as some compiler flag warned about this.